### ISSUE TITLE
UnhandledPromiseRejectionWarning

### ISSUE THREAD

**Issue by oalders** [lannonbr/issue-label-manager-action#10]
I ran into this here: https://github.com/oalders/changelog-generator/pull/1/checks?check_run_id=242518473

It was supposed to set up 3 default labels.  Two of them already existed, albeit with different letter casing.  The delete worked, but only one label `Hacktoberfest` existed.  `Bug` and `Enhancement` did not get created.

```
Run lannonbr/issue-label-manager-action@master
[Action] Deleting Label: bug
[Action] Deleting Label: duplicate
[Action] Deleting Label: enhancement
[Action] Deleting Label: good first issue
[Action] Deleting Label: help wanted
[Action] Deleting Label: invalid
[Action] Deleting Label: question
[Action] Deleting Label: wontfix
[Action] Creating Label: Bug
[Action] Creating Label: Enhancement
[Action] Creating Label: Hacktoberfest
(node:2745) UnhandledPromiseRejectionWarning: HttpError: Validation Failed
    at /home/runner/work/_actions/lannonbr/issue-label-manager-action/master/lib/index.js:86:941
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async /home/runner/work/_actions/lannonbr/issue-label-manager-action/master/lib/index.js:202:1214
(node:2745) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:2745) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:2745) UnhandledPromiseRejectionWarning: HttpError: Validation Failed
    at /home/runner/work/_actions/lannonbr/issue-label-manager-action/master/lib/index.js:86:941
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async /home/runner/work/_actions/lannonbr/issue-label-manager-action/master/lib/index.js:202:1214
(node:2745) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
```

**Comment by citycide** [comment]
I think this may be the same issue as mine, #9.

**Comment by crazy-max** [comment]
Can you test with my [labeler action](https://github.com/crazy-max/ghaction-github-labeler)? Maybe it's an issue with octokit.

**Comment by citycide** [comment]
@crazy-max your implementation works.

**Comment by schalkneethling** [comment]
For me, it seemed to create all the labels, but I also see the same failures.

**Comment by schalkneethling** [comment]
@oalders I am wondering if the problem could be related to this line:
https://github.com/lannonbr/issue-label-manager-action/blob/4766b75abab314a7c9de12dba95b407a92187782/index.js#L91

When comparing strings or characters, this function is case-sensitive so, `infra` and `Infra` will not match. Could that perhaps cause an error to be thrown in https://github.com/lannonbr/issue-label-manager-action/blob/4766b75abab314a7c9de12dba95b407a92187782/index.js#L32, as it will be set to `create` instead of `update`.

**Comment by oalders** [comment]
@schalkneethling I actually don't know. I opened this issue a few years ago and kind of moved on to other things after I hit this problem.

**Comment by schalkneethling** [comment]
> @schalkneethling I actually don't know. I opened this issue a few years ago and kind of moved on to other things after I hit this problem.

I reckon I will open a pull request and see where it goes from there. Thank you for getting back to me.

**Comment by lannonbr** [comment]
This should now be resolved as of release 3.0.1
